index.html

```diff
@@ -6,6 +6,37 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-
+<div class="container">
+    <section class="item item1">
+        A
+    </section>
+    <section class="item item2">
+        B: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci aut beatae eum eveniet maxime modi
+        obcaecati possimus quidem saepe vero! Eos esse exercitationem facilis quasi reiciendis? Ab asperiores aut
+        doloribus eligendi, possimus quisquam recusandae reiciendis rerum sit vel. Accusamus alias at beatae corporis
+        dolorem ea excepturi facere ipsum laudantium mollitia pariatur porro, quo repudiandae soluta unde, vitae
+        voluptatum. Accusantium dolorum earum eligendi hic minus nemo, neque, praesentium quibusdam recusandae rem
+        temporibus ut voluptas voluptatum? Accusamus corporis explicabo illum minus perspiciatis. Aliquid doloremque
+        fuga impedit, natus odio perspiciatis quae qui sapiente vero voluptas? Aspernatur illum provident sit! Dolorem,
+        ipsum officiis! Ab accusamus accusantium atque aut blanditiis deleniti dignissimos, earum eius eligendi enim est
+        ex fuga hic illum ipsam iste iusto laborum laudantium magni modi pariatur perferendis quae quis quisquam
+        reiciendis sed similique tempore vel veniam voluptatum? Amet eligendi illum, ipsam iure iusto porro quasi
+        voluptatem? Asperiores assumenda porro unde. Enim, quo.
+    </section>
+    <section class="item item3">
+        C
+    </section>
+    <section class="item item4">
+        D
+    </section>
+    <section class="item item5">
+        E
+    </section>
+    <section class="itemleeg">
```

Comment:
je hoef geen aparte section voor een lege gedeelte te maken. Je kan met alle gerust regel 35 en 36 van je code weghalen. Wel in de grid vermelden lege grid-cel met een punt '.' Zie een voorbeeld onderaan: i.p.v. 'x' gebruik je '.')
grid-template-areas:
            "a c c"
            "b c c"
            "b e d"
            "b e ."
            "f f f";